src/shim.c

```diff
@@ -336,6 +336,28 @@ static unsigned char shim_nbuf_get_priority(void *nbuf)
 	return skb->priority;
 }
 
+static unsigned char shim_nbuf_get_chksum_done(void *nbuf)
+{
+	struct sk_buff *skb = (struct sk_buff *)nbuf;
+
+	if (!skb) {
+		pr_err("%s: Unable to get checksum\n", __func__);
+		return -1;
+	}
+
+	return skb->csum_complete_sw;
```

Comment:
In general this flag needs additional data, has this been tested?

```
  skb->csum = csum;
  skb->ip_summed = CHECKSUM_COMPLETE;
  skb->csum_complete_sw = 1;
  skb->csum_valid = !sum;
```

Reply:
I have tested linux driver with RPI4 with this change. It's building and also connecting with AP.

Reply:
With CSUM offload enabled and using IPv4 fragmentation?

Reply:
Yes, need to add those flags.